server/src/server.js

```diff
@@ -255,9 +263,14 @@ app.get("/shots", function (req, res) {
   if (! req.deviceId) {
     return simpleResponse(res, "You must have the addon installed to see your shot index", 403);
   }
+  let simple = shouldRenderSimple(req);
```

Comment:
Why bother calling this here? I think you should do it inside the if on line 269